easybuild/easyconfigs/m/MAGeCK-VISPR/MAGeCK-VISPR-0.5.6-foss-2023a.eb

```diff
@@ -0,0 +1,74 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>; Guillermo AR Piccoli <guillermo.piccoli@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'PythonBundle'
+
+name = 'MAGeCK-VISPR'
+version = '0.5.6'
+
+homepage = 'https://bitbucket.org/liulab/mageck-vispr/'
+description = """MAGeCK-VISPR is a comprehensive quality control, analysis and visualization workflow for CRISPR/Cas9
+  screens The workflow combines the MAGeCK algorithm to identify essential genes from CRISPR/Cas9 screens considering
+  multiple conditions with VISPR to interactively explore results and quality control in a web-based frontend."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+dependencies = [
+    ('SciPy-bundle', '2023.07'),
+    ('Meson', '1.4.0')
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('PyYAML', '5.4.1', {
+        'modulename': 'yaml',
+        'checksums': ['607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e'],
+    }),
+    ('pyproject-metadata', '0.7.1', {
+        'checksums': ['0a94f18b108b9b21f3a26a3d541f056c34edcb17dc872a144a15618fed7aef67'],
+    }),
```

Comment:
this looks like something intended to be picked up as a dependency for meson-python, so, i would say just use meson-python-0.15.0-GCCcore-12.3.0.eb. Seeing this, I now wonder if this was the only reason for updating Meson anyway.. which means that adding it wasn't actually needed at all and it just stems from this.